### PR TITLE
1335 - fixing paper documents

### PR DESCRIPTION
**Pull request by codyseibert**
- refactoring to no longer store receivedAt inside the createdAt property; instead, creating a new receivedAt property

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/presenter/computeds/formattedWorkQueue.js [hunk 1]
```diff
@@ -9,6 +9,16 @@ import { state } from 'cerebral';
 import _ from 'lodash';
 import moment from 'moment';
 
+const isDateToday = (date, applicationContext) => {
```
Comment (matthopson) [inline web-client/src/presenter/computeds/formattedWorkQueue.js:12]: Does this also need the "yesterday" logic?

Comment (matthopson) [reply web-client/src/presenter/computeds/formattedWorkQueue.js:12]: https://github.com/flexion/ef-cms/pull/2153/files#diff-051a76dce2615e2489552f4f7bb88807R17

Comment (matthopson) [reply web-client/src/presenter/computeds/formattedWorkQueue.js:12]: nvm, I see what's going on further down

**Review by matthopson — APPROVED** [review]

**Pull request merged**